recipe/0026-toc-opera/manifest.json

```diff
@@ -0,0 +1,465 @@
+{
+  "@context": [
+    "http://www.w3.org/ns/anno.jsonld",
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "id": "https://example.org/iiif/cookbook-recipes/0026-toc-opera/manifest",
+  "type": "Manifest",
+  "label": { "it": [ "L'elisir d'amore" ], "en": [ "The elixir of love" ] },
+  "items": [
+    {
+      "id": "https://example.org/iiif/cookbook-recipes/0026-toc-opera/manifest/canvas/1",
+      "type": "Canvas",
+      "width": 1920,
+      "height": 1080,
+      "duration": 7278.422,
+      "items": [
+        {
+          "id": "https://example.org/iiif/cookbook-recipes/0026-toc-opera/manifest/canvas/1/annotation_page/1",
+          "type": "AnnotationPage",
+          "items": [
+            {
+              "id": "https://example.org/iiif/cookbook-recipes/0026-toc-opera/manifest/canvas/1/annotation_page/1/annotation/1",
+              "type": "Annotation",
+              "motivation": "painting",
+              "target": "https://example.org/iiif/cookbook-recipes/0026-toc-opera/manifest/canvas/1",
+              "body": {
+                "id": "https://iiif-av.s3.amazonaws.com/examples/indiana/donizetti-elixir/vae0637_accessH264_low.mp4",
+                "type": "Video",
+                "format": "video/mp4",
+                "height": 1080,
+                "width": 1920,
+                "duration": 7278.422
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "structures": [
+    {
+      "type": "Range",
+      "id": "https://example.org/iiif/cookbook-recipes/0026-toc-opera/manifest/range/r9ad47417-1344-4991-a41a-b084c69a12d4",
+      "label": {
+        "en": [
+          "IU Opera Theatre"
+        ]
+      },
+      "items": [
```

Comment:
While a real world scenario, do we need such an extensive structure for a recipe? Can this be elided without losing the point of the ToC?